content/docs/config-files.md

```diff
@@ -101,7 +110,7 @@ The full list of Scintilla messages for `type=0` and `type=1` Scintilla messages
 
 The `wParam` command IDs for `type=2` Notepad++ messages can be found as the `IDM` constants in the source file [menuCmdID.h](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/menuCmdID.h), or you can look at the `localization\English.xml` (or your language of choice), which lists the `<Item id="...">` next to the text of the command; the value of the `id` attribute is the "command ID".
 
-For `type=3` search-and-replace macros, see the detailed description in ["Searching > Searching actions when recorded as macros"](../searching/#searching-actions-when-recorded-as-macros).
+For `type=3` search-and-replace macros, see the detailed description in ["Searching > Searching actions when recorded as macros"](../searching/#searching-actions-when-recorded-as-macros).  The message numbers for `type=3` come from [FindReplaceDlg_rc.h](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/ScitillaComponent/FindReplaceDlg_rc.h).

```

Comment:
I don't see the correlation between "type 3" and this added line.

Reply:
I will try to make that more clear.

Reply:
Actually, it's not really helpful, so I took it out again.